.vscode/tasks.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "command": "dotnet",
+            "type": "process",
+            "args": [
+                "build",
+                "${workspaceFolder}/src/API/Loggy.Api/Loggy.Api.csproj"
+            ],
+            "problemMatcher": "$msCompile"
+        }
+    ]
+}
```

Comment:
Okay cool so is this used ... by minikube?

Reply:
Nope, still just auto-generated VSCode things. Should really have gone in a separate PR.